Monika After Story/game/script-farewells.rpy

```diff
@@ -947,6 +947,25 @@ label bye_going_somewhere:
         $ _history_list.pop()
         m 1lksdlc "..."
         m 1eksdlb "What am I saying? Of course I'll go with you!"
+        if mas_isMoniLove(higher=True):
+        m 1hub "Oh, okay!"
+        menu:
+            m 3tub "Where are you taking me?"
+            "School.":
+                m 1eka "Ahaha [Player], I cant wait to see all your friends!"
```

Comment:
Monika can't really see your friends, so I think this is a bit of a strange thing to say.